src/mopi/blocks/io.py

```diff
@@ -50,3 +57,18 @@ def pickle_saving(
     )
     with open(path + f"/{id}.pkl", "wb") as f:
         joblib.dump(object, f, compress=9)
+
+
+def export_pipeline(name: str, pipeline: "Pipeline") -> None:
+    if os.path.exists(Const.output_pipelines_path) is False:
+        os.makedirs(Const.output_pipelines_path)
+
+    with open(f"{Const.output_pipelines_path}/{name}", "wb") as handle:
+        pickle.dump(pipeline, handle, protocol=pickle.HIGHEST_PROTOCOL)
+
+
+def load_pipeline(name: str) -> "Pipeline":
+    with open(f"{Const.output_pipelines_path}/{name}", "rb") as handle:
+        pipeline = pickle.load(handle)
+
+    return pipeline
```

Comment:
I chose to write a new function for this instead of reusing the above functions. They can be fused later on, but I wanted to make sure that I don't mess up something in the existing pipeline with the new functionality.